### ISSUE TITLE
ValueError: numpy.ndarray size changed

### ISSUE THREAD

**Issue by daniiltroshkov** [Deeplite/deeplite-torch-zoo#102]
Hi!

I have installed `deeplite-torch-zoo` following the instructions from here:

https://docs.deeplite.ai/neutrino/zoo.html#zoo-available-models

I did:

```
pip3 install --upgrade pip
pip3 install deeplite-torch-zoo
```

If I try to import deeplite_torch_zoo.wrappers.wrapper I've got an error:

```
Traceback (most recent call last):
  File "dl_model.py", line 1, in <module>
    from deeplite_torch_zoo.wrappers.wrapper import (
  File "/home/daniil/.local/lib/python3.8/site-packages/deeplite_torch_zoo/wrappers/__init__.py", line 1, in <module>
    from .wrapper import *
  File "/home/daniil/.local/lib/python3.8/site-packages/deeplite_torch_zoo/wrappers/wrapper.py", line 1, in <module>
    from deeplite_torch_zoo.wrappers.datasets import *
  File "/home/daniil/.local/lib/python3.8/site-packages/deeplite_torch_zoo/wrappers/datasets/__init__.py", line 1, in <module>
    from .classification import *
  File "/home/daniil/.local/lib/python3.8/site-packages/deeplite_torch_zoo/wrappers/datasets/classification/__init__.py", line 2, in <module>
    from .vww import *
  File "/home/daniil/.local/lib/python3.8/site-packages/deeplite_torch_zoo/wrappers/datasets/classification/vww.py", line 2, in <module>
    import pyvww
  File "/home/daniil/.local/lib/python3.8/site-packages/pyvww/__init__.py", line 1, in <module>
    import pyvww.pytorch
  File "/home/daniil/.local/lib/python3.8/site-packages/pyvww/pytorch/__init__.py", line 1, in <module>
    from .datasets import VisualWakeWordsClassification
  File "/home/daniil/.local/lib/python3.8/site-packages/pyvww/pytorch/datasets.py", line 5, in <module>
    from pyvww.utils import VisualWakeWords
  File "/home/daniil/.local/lib/python3.8/site-packages/pyvww/utils/__init__.py", line 1, in <module>
    from .visualwakewords import VisualWakeWords
  File "/home/daniil/.local/lib/python3.8/site-packages/pyvww/utils/visualwakewords.py", line 4, in <module>
    from pycocotools.coco import COCO
  File "/home/daniil/.local/lib/python3.8/site-packages/pycocotools/coco.py", line 52, in <module>
    from . import mask as maskUtils
  File "/home/daniil/.local/lib/python3.8/site-packages/pycocotools/mask.py", line 3, in <module>
    import pycocotools._mask as _mask
  File "pycocotools/_mask.pyx", line 1, in init pycocotools._mask
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
```

It helps to do:

```
pip3 install --upgrade numpy
```

But I've got error here:

```
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: numpy in /home/daniil/.local/lib/python3.8/site-packages (1.21.4)
Collecting numpy
  Using cached numpy-1.22.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (16.8 MB)
Installing collected packages: numpy
  Attempting uninstall: numpy
    Found existing installation: numpy 1.21.4
    Uninstalling numpy-1.21.4:
      Successfully uninstalled numpy-1.21.4
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
deeplite-torch-zoo 1.2.2 requires numpy==1.21.4, but you have numpy 1.22.1 which is incompatible.
Successfully installed numpy-1.22.1
```

If I ignore this ERROR I can successufly run my script after that.

Here is my script. It is just downloding model from zoo:

```
from deeplite_torch_zoo.wrappers.wrapper import (
    get_model_by_name, get_data_splits_by_name)
model = get_model_by_name(
    model_name="resnet18",
    dataset_name="cifar100",
    pretrained=True,
    progress=True,
    device="cpu")
```

I assigning this issue to Yasser, but not 100% sure if he is mantainer of this things. Feel free to reassign this issue.

Thank you!

**Comment by ivan-lazarevich** [comment]
@yasseridris we had to force [different numpy versions for different python versions](https://github.com/Deeplite/deeplite-torch-zoo/blob/bad382bf640a8e89fee9ee28042bbe1a78d30a74/setup.py#L40) because of `pycocotools` package incompatibility.

If using different numpy versions is not desirable, I can try downgrading the pycocotools dep version.

**Comment by yasseridris** [comment]
@ivan-lazarevich No that's fine.. Now that we have multiple python distributions so there shouldn't be an issue.. We just have to make sure that the numpy version installed for a specific python interpreter works with all ubuntu releases we claim support for (Ubuntu>=16.04)

**Comment by yasseridris** [comment]
@daniiltroshkov Please include the information about the python and ubuntu versions.

**Comment by daniiltroshkov** [comment]
```
uname -a
Linux jenkins 5.11.0-46-generic #51~20.04.1-Ubuntu SMP Fri Jan 7 06:51:40 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
python3 --version
Python 3.8.10
```

**Comment by ivan-lazarevich** [comment]
@yasseridris @daniiltroshkov update: updated deps to`pycocotools==2.0.4` which seems to have resolved issues with `numpy==1.19.5` so that numpy version is common now for all python versions.
